ddl/db_test.go

```diff
@@ -1659,6 +1659,76 @@ func (s *testDBSuite) TestCreateTableWithLike(c *C) {
 	s.tk.MustExec("drop database ctwl_db1")
 }
 
+func (s *testDBSuite) TestCreateTableWithLike2(c *C) {
+	s.tk = testkit.NewTestKit(c, s.store)
+	s.tk.MustExec("use test_db")
+	s.tk.MustExec("drop table if exists t1,t2;")
+	defer s.tk.MustExec("drop table if exists t1,t2;")
+	s.tk.MustExec("create table t1 (a int, b int, c int, index idx1(c));")
+
+	t1Info := testGetTableByName(c, s.s, "test_db", "t1")
```

Comment:
s/`t1Info`/`tbl1` ?
The result is a `table.Table` type instead of `TableInfo`. I think `tbl1` is better.
And `checkTableT2Info` can be renamed to `checkTbl2`

Reply:
done.